grizzly/common/status_reporter.py

```diff
@@ -529,51 +529,52 @@ class _TableFormatter:
 
     def __init__(
         self,
-        columns: Tuple[str, ...],
-        formatters: Tuple[Optional[Callable[..., str]]],
+        column_names: Tuple[str, ...],
+        formatters: Tuple[Optional[Callable[..., str]], ...],
         vsep: str = " | ",
         hsep: str = "-",
     ) -> None:
         """Initialize a TableFormatter instance.
 
         Arguments:
-            columns: List of column names for the table header.
+            column_names: List of column names for the table header.
             formatters: List of format functions for each column.
                         None will result in hiding that column.
             vsep: Vertical separation between columns.
             hsep: Horizontal separation between header and data.
         """
-        assert len(columns) == len(formatters)
+        assert len(column_names) == len(formatters)
         self._columns = tuple(
-            column for (column, fmt) in zip(columns, formatters) if fmt is not None
+            column for (column, fmt) in zip(column_names, formatters) if fmt is not None
         )
-        self._formatters = formatters
+        self._formatters = tuple(formatters)
```

Comment:
Is `tuple()` needed here? The type hint says `formatters` is already a `tuple`.

Reply:
Nope, it isn't. I initially had changed the signature to `Iterable` but that added errors because you can't `len()` an `Iterable`.